s3tests_boto3/functional/test_iam.py

```diff
@@ -0,0 +1,198 @@
+import json
+from operator import eq
+
+from botocore.exceptions import ClientError
+from nose.plugins.attrib import attr
+
+from s3tests.functional.utils import assert_raises
+from . import get_tenant_iam_client, get_tenant_user_id
+from .utils import _get_status
+
+
+@attr(resource='user-policy')
+@attr(method='put')
+@attr(operation='Verify Put User Policy')
+@attr(assertion='succeeds')
+@attr('user-policy')
+def test_put_user_policy():
```

Comment:
I suggest to store a configurable yaml or property file which will map test function name to internal test ids. Looking for test ids from code would be easier for rechecking.

Reply:
Or `@attr(id= ` can also be instrumented.